.github/workflows/deploy-feature-envs.yaml

```diff
@@ -70,13 +77,16 @@ jobs:
 
       - name: Push FE aleph-node image to the feature-env-aleph-node registry
         env:
-          IMAGE_TAG: ${{ env.FE_IMAGETAG_PREFIX }}${{ steps.get_branch.outputs.branch_imagetag_full }}
+          IMAGE_TAG:
```

Comment:
again, is that ok without any `\`, `|` or `>`?

Reply:
Yes